homeassistant/components/izone/climate.py

```diff
@@ -54,6 +58,20 @@
     Controller.Fan.AUTO: FAN_AUTO,
 }
 
+ATTR_AIRFLOW = "airflow"
+
+IZONE_SERVICE_AIRFLOW_MIN = "airflow_min"
+IZONE_SERVICE_AIRFLOW_MAX = "airflow_max"
+
+IZONE_SERVICE_AIRFLOW_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_ENTITY_ID): cv.entity_id,
```

Comment:
`async_register_entity_service` already takes care of `ENTITY_ID`, besides, a user could be targeting an area or device as well.

See: <https://developers.home-assistant.io/docs/dev_101_services#entity-services>

Reply:
Thanks. So I can remove line 68?

Should I open a new PR for this change or add to this one?

Reply:
The variable passed to the entity service helper should just be a dict with voluptuous validators and the validators should not include `CONF_ENTITY_ID`. See the example in the linked docs above.

Please open a new PR. Thanks!